apps/data-types.md

```diff
@@ -640,3 +640,22 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | productNumber | string | The product number. |
 | watchers | number | The number of watchers. |
 | links | array of [link](page:apps-data-types#link) | The link to the watched product. |
+
+## webhook
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| id | string | The unique identifier of the webhook. |
+| active | boolean | The status of the webhook. Can be `true` or `false`. |
+| eventType | string | The event that triggers the webhook. |
+| callbackUrl | string | The URL to your application where the event is received. |
+| createdOn | string | The date/time the webhook was created. Expressed according to ISO 8601. Example: `2017-03-14T08:42:49.000Z` |
+| updatedOn | string | The date/time the webhook was updated. Expressed according to ISO 8601. Example: `2017-03-15T17:12:15.000Z` |
+| links | array of [link](page:apps-data-types#link) | The link to the webhook resource of the shop. |
+
+## webhook (create request)
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| callbackUrl | string | The URL to the application where the event is received.|
```

Comment:
Maybe mention, that the URL has to be a "https" url. Same for scriptUrl of the scriptTag (create request)